### DISCUSSION TITLE
Brower support like v6 in v7

### DISCUSSION THREAD

**Discussion by juliuslissau** [mantinedev/mantine#5200] [Feature requests]
Only supporting Chrome version >108 in v7 (which is only one year old) is very unfortunate.
Would be great if the browser support from v6 could still be supported in v7.

I only discovered today that this was the case, because a user called me and said they couldn't scroll down on a modal.

The issue turned out to be that CSS `100dvh` (dynamic view high (and similar)) is only supported in Chrome v108+. 
I could overwrite this with CSS for my modals to return to `100vh`, but not sure what other issues might arise. 

A lot of users is getting left out by this small CSS change, which doesn't even add a lot of value in my opinion: https://caniuse.com/mdn-css_types_length_viewport_percentage_units_dynamic

(I don't know if there's a list of somewhere or a main reason Mantine v7 couldn't keep the same browser support as v6?)

---

Update: I solved my issue regarding the Modal by adding this code to my theme object.
This will overwrite the places where Mantine v7 uses dvh, and just replace it with vh which has way better browser support and gives roughly the same look.

```
Modal: {
  vars: (theme, props) => {
    if (props.fullScreen === true) {
      return {
        content: {
          "--_content-height": "100vh",
          "--modal-y-offset": "5vh",
        },
      };
    }
  },
  styles: {
    content: {
      "maxHeight":
        "var(--_content-max-height, calc(100vh - var(--modal-y-offset) * 2))",
    },
  }
}
```

**Comment by rtivital** [comment]
Supported version is a bare minimum that all Mantine components support. Most of the components work in older browsers as is. If you need to support older browsers, you can override these CSS rules on your side – since there aren't many of them, it is not hard.

**Reply by juliuslissau** [reply]
True, I appreciate that as well.

So the bump in browser support for v7, is only because of using newer CSS rules? No JS problems or other reasons?

Even so, it can be tricky to spot which components now suddenly have changed to using these modern CSS rules, when you have a large code base. Would be awesome if those kind of "breaking changes" could be mentioned as well in the Changelog or on component pages.

Anyway, thanks so much for all your work Vitaly. I know I'm not entitled to demand anything for open source software. Just sharing my thoughts and struggles using Mantine if you find them useful.

**Reply by rtivital** [reply]
Yes, as far as I know, there aren't any new JS features used. If you find bugs with older browser compatibility, you are welcome to open an issue, and we will figure out how to fix them. I do not have spare capacity to test every component in older browsers, so the compatibility table includes worst-case support for each component.

**Comment by 7iomka** [comment]
@juliuslissau
I think we need some workaround in mantine library that uses @supports to choose which version can use, dvh or vh.
If you have free time, you are welcome to open PR with this changes.
Dvh fixes issues with safari browsers on iOS, but of course the rest world need the working fallback.
